db/migrate/20161109150714_add_templates_to_resolve_url.rb

```diff
@@ -0,0 +1,21 @@
+class AddTemplatesToResolveUrl < ActiveRecord::Migration[5.0]
```

Comment:
I've added a spec that directly calls the up method of the migration.  I can imagine running all the past migrations in order may affect other specs get easily broken in the future when many migrations are added after this one.